bool attach_to_table
int32 table_number
---
bool complete